Helper/Cart.php

```diff
@@ -1282,41 +1290,24 @@ function ($item) use ($imageHelper, &$totalAmount, &$diff, $storeId, $currencyCo
                 // Get product description and image
                 ////////////////////////////////////
                 $product['description'] = strip_tags($_product->getDescription());
-                $variantProductToGetImage = $_product;
 
-                // This will override the $_product with the variant product to get the variant image rather than the main product image.
                 try {
-                    // If the cart item is type of bundle product, its SKU is a combination with bundle selections,
-                    // so we need to retrieve the sku of bundle product without bundle selections.
-                    $product_sku = 'bundle' == $item->getProductType()
-                                   ? $_product->getData('sku')
-                                   : $product['sku'];
-                    $variantProductToGetImage = $this->productRepository->get($product_sku, false, $storeId);
+                    $productToGetImage = $this->itemResolver->getFinalProduct($item);
```

Comment:
Would this pick the correct variant image?

Reply:
Yes, right

Reply:
@johnmccombs1 Can we use $productToGetImage in all cases when we use $_product for now?
It's not clear what's the difference and we can improve readability if use only one product object.

If you don't see any problem with this could you please create the task for this refactoring.

Reply:
@vitaliyreznikov No, the $productToGetImage is different with $_product. Below is the logic to get $productToGetImage https://prnt.sc/ttrupo I don't think we can use it in all cases